### PR TITLE
Shape Detection API post

### PR DESCRIPTION
**Pull request by tomayac**
What's changed, or what was fixed?
- Adds an Update post for the Shape Detection API.

**Target Live Date:** 2019-01-16

- [x] This has been reviewed and approved by (@petele)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2019/01/shape-detection.md [hunk 1]
```diff
@@ -180,6 +181,20 @@ try {
 }
 ```
 
+## Feature detection {: #featuredetection}
+
+Purely checking for the existence of the constructors to feature detect the Shape Detection
+API doesn’t suffice, as Chrome on Linux and Chrome OS currently still exposes the detectors, but
```
Comment (beaufortfrancois) [inline src/content/en/updates/2019/01/shape-detection.md]: Nit : `expose`

File: src/content/en/updates/2019/01/shape-detection.md [hunk 2]
```diff
@@ -180,6 +181,20 @@ try {
 }
 ```
 
+## Feature detection {: #featuredetection}
+
+Purely checking for the existence of the constructors to feature detect the Shape Detection
```
Comment (beaufortfrancois) [inline src/content/en/updates/2019/01/shape-detection.md]: I would add a note saying this is temporary.

File: src/content/en/updates/2019/01/shape-detection.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Shape Detection API allows for faces, barcodes, and text to be detected in
+images.
```
Comment (petele) [inline src/content/en/updates/2019/01/shape-detection.md]: `description` must all be on one line, no line breaks.

File: src/content/en/updates/2019/01/shape-detection.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+
+{# wf_updated_on: 2019-01-14 #}
+{# wf_published_on: 2019-01-07 #}
+{# wf_tags: capabilities,shape-detection,progressive-web-apps,webapp #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: The Shape Detection API allows for faces, barcodes, and text to be detected
+in images. #}
```
Comment (petele) [inline src/content/en/updates/2019/01/shape-detection.md]: `wf_featured_snippet` must be all on one line, no line breaks.

**Pull request merged**